index.html

```diff
@@ -1037,9 +1037,9 @@ <h3>Identifiers</h3>
         <p>
 When expressing statements about a specific thing, such as a person, product,
 or organization, it is often useful to use some kind of identifier so that
-others can express statements about the same thing. Identifiers that others
-are expected to use when expressing statements about a specific thing MUST be
-expressed using the <code>id</code> <a>property</a>.
+others can express statements about the same thing. This specification defines
+the <code>id</code> <a>property</a> for identifiers. This allows for the
```

Comment:
This is a normative change as you removed the MUST language. Clearly, other identifiers can be used and associated with other properties. We should probably change it such that the normative requirement is "about a specific thing that has associated properties". We never intended to say you MUST use `id`... rather, if you think you're going to make statements about the same subject as someone else, you MUST use `id`.

What about this:

The <code>id</code> <a>property</a> MUST be used when statements about a particular subject are expected to be expressed by more than one entity.

Reply:
My contention is that this "normative" statement is unenforceable in a data model spec and therefore a mistake in need of the suggested editorial change.

Reply:
This is the one inappropriately "normative" statement left in the spec (the rest were edited out in PR #443, but this one was missed). 
This one should be changed for the same reason the rest of them were: we have no protocol to to enforce how the `id` property must be used, so the spec shouldn't say how it must be used.

Reply:
We have been around this discussion several times before. I think we have agreed that there are two types of conformance statement about properties:

1.    Properties that MAY be present, but if present MUST have a specific format, and
2.    Properties that MUST be present, and MUST have a specific format.

The contentious issue appears to be about properties of type 1. How do we stop someone defining a property that is semantically the same as  a type 1 property but has a different name and different format? I thought that in Barcelona we agreed that we cannot stop people from doing this. In fact this issue also applies to properties of type 2 as well. For example, we cannot stop someone defining a DateOfIssuance property with a format that is not RFC 3999. 

So for the id property, all we can do is:

a. define what its semantics and name are
b. say it MAY be present in objects
c. say that it MUST be a URL.

This leads to the following much simplified text:

"The id property is defined to unambiguously refer to an object such as a person, product, or organization. This is so that others can express statements about the same thing. Objects that others can refer to, MAY contain the id property. "

Reply:
please see changes in [4dcd022](https://github.com/w3c/vc-data-model/pull/534/commits/4dcd0221699298fa6e9ae1ab75888438468645e9)